modules/utils/validation.js

```diff
@@ -86,12 +86,18 @@ function validateRule (item, path, itemId, meta, c) {
   const {showErrorMessage} = config.settings;
   let id = item.get("id");
   let properties = item.get("properties");
-  let field = properties.get("field") || null;
+  // let field = properties.get("field") || null;
```

Comment:
Why is this commented out

Reply:
This is leftover, `field` declaration has been changed and moved few lines below